packages/components/atoms/f-card/src/components/Card.vue

```diff
@@ -92,32 +101,27 @@ $card--pageContentWrapper-width           : 472px; // so that it falls on our 8p
     .c-card--pageContentWrapper {
         width: 100%;
         transition: 250ms padding ease-in-out;
-        padding-left: 6%;
-        padding-right: 6%;
         margin: spacing(x5) 0;
 
-        @include media('>=narrow') {
-            padding-left: 10%;
-            padding-right: 10%;
-        }
-
         @include media('>=#{$card--pageContentWrapper-width}') {
             width: $card--pageContentWrapper-width;
             margin: spacing(x5) auto;
-            padding-left: spacing(x10);
-            padding-right: spacing(x10);
         }
-    }
 
-    .c-card-heading {
-        margin-bottom: spacing(x2);
-    }
+        .c-card--innerSpacing {
```

Comment:
can we remove the css nesting here? less nesting - the better.

Reply:
This one is nested for a reason. `.c-card--pageContentWrapper` modifies `.c-card` and has different inner spacing.